tests/base/src/main/resources/schema/elytron-client-1_4.xsd

```diff
@@ -976,6 +980,56 @@
         </xsd:attribute>
     </xsd:complexType>
 
+    <xsd:complexType name="masked-password-type">
+        <xsd:annotation>
+            <xsd:documentation>
+                A masked password definition.
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:attribute name="algorithm" type="xsd:string" default="masked-MD5-DES">
+            <xsd:annotation>
+                <xsd:documentation>
+                    An algorithm encrypting the password.
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+        <xsd:attribute name="key-material" type="xsd:string" default="somearbitrarycrazystringthatdoesnotmatter">
+            <xsd:annotation>
+                <xsd:documentation>
+                    A key encrypting the password.
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+        <xsd:attribute name="iteration-count" type="xsd:positiveInteger" use="required">
+            <xsd:annotation>
+                <xsd:documentation>
+                    A count of iterations used to encrypt the password.
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+        <xsd:attribute name="salt" type="xsd:string" use="required">
+            <xsd:annotation>
+                <xsd:documentation>
+                    A salt used to encrypt the password.
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+        <xsd:attribute name="masked-password" type="xsd:string" use="required">
+            <xsd:annotation>
+                <xsd:documentation>
+                    The base64 encrypted password. (without the "MASK-" prefix)
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+        <xsd:attribute name="initialization-vector" type="xsd:string" use="optional">
+            <xsd:annotation>
+                <xsd:documentation>
+                    An initialization vector used to encrypt the password.
```

Comment:
Maybe "The initialization vector that was used to encrypt the password."

Reply:
fixed